internal/contract/controlplane.go

```diff
@@ -76,6 +76,13 @@ func (c *ControlPlaneContract) Replicas() *Int64 {
 	}
 }
 
+// NodeDrainTimeout provides access to the nodeDrainTimeout of a ControlPlane.
+func (c *ControlPlaneContract) NodeDrainTimeout() *Duration {
```

Comment:
Can be removed as we now use `contract.ControlPlane().MachineTemplate().NodeDrainTimeout()` instead